dnf-plugins-core.spec

```diff
@@ -1,4 +1,5 @@
-%{?!dnf_version: %global dnf_version 0.6.5}
+%{?!dnf_lowest_compatible: %global dnf_lowest_compatible 0.6.5}
+%{?!dnf_not_compatible: %global dnf_not_compatible 0.7.0}
```

Comment:
This is not true. If we are going to implement semantic versioning, the first possible incompatible version is 1.0.
See: http://semver.org/